### PR TITLE
Create user profile screen UI

### PR DESCRIPTION
**Pull request by hiepdnh**
https://edu-redmine.framgia.vn/issues/6292
device-2019-02-18-151942

### CODE REVIEW

**Comment by hiepdnh** [comment]
ready

File: app/src/main/java/com/framgia/chat_03/screen/home/HomeActivity.java [hunk 1]
```diff
@@ -77,7 +81,8 @@ public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
                 fragment = FriendFragment.newInstance();
                 break;
             case R.id.navigation_profile:
-                return true;
+                fragment= ProfileFragment.newInstance();
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/chat_03/screen/home/HomeActivity.java]: format code

File: app/src/main/java/com/framgia/chat_03/screen/profile/ProfileFragment.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
+                             @Nullable Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.fragment_profile, container, false);
+    }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/chat_03/screen/profile/ProfileFragment.java]: remove if unused

File: app/src/main/res/layout/fragment_profile.xml [hunk 3]
```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/color_white">
+
+
```
Comment (DoanVanToan) [inline app/src/main/res/layout/fragment_profile.xml]: remove bl

File: app/src/main/res/layout/fragment_profile.xml [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+        android:textColor="@color/color_white"
+        android:textSize="@dimen/sp_18"
+        app:layout_constraintEnd_toStartOf="@+id/text_user_phone"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintHorizontal_chainStyle="packed"
+        app:layout_constraintStart_toEndOf="@+id/image_background"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/text_email" />
+
+
```
Comment (DoanVanToan) [inline app/src/main/res/layout/fragment_profile.xml]: remove bl

**Review by DoanVanToan — APPROVED** [review]

**Pull request merged**